.circleci/workflows.yml

```diff
@@ -1024,12 +1506,12 @@ workflows:
 
   sonarcloud_analysis_front:
```

Comment:
Can't see conditions to avoid execution on Release, DockerImage build & RPM build

Reply:
OK, because it is linked to another issue :)